content/blog/2018-09-14-react-native-boilerplate.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "React-Native Typescript Boilerplate"
+sub_title: "Build cross-platform mobile apps faster."
+tags: ["React Native", "TypeScript", "Mobile"]
+date: 2018-09-14T09:44:38-07:00
+publishdate: 2018-09-14T09:44:38-07:00
+hero_image: /images/blog/2018-09-14-react-native-boilerplate/hero.png
+author:
+  name: Reid Weber
+  url: https://medium.com/@reidweb1
+  mail: reid@meshstudio.io
+  avatar: https://avatars2.githubusercontent.com/u/8824846?s=460&v=4
+---
+
+React Native has established itself as an incredibly powerful tool for building cross-platform mobile applications. We set out to try and create a clone-able repository so developers can jump right into building features/UI instead of spending hours configuring their environment.
+
+
+### Getting Started
+
+
+If you want to jump right into developing your own cross-platform mobile application, get started with the README [here](https://github.com/meshhq/react-native-boilerplate).
+
+If you want more insight into the tooling/frameworks used in this project - continue reading.
+
+First go ahead and clone the repository so you can follow along in the codebase.
+
+```
+git clone git@github.com:meshhq/react-native-boilerplate.git
+```
+
+
+## Tooling
+
+
+> [`react-native-cli`](https://facebook.github.io/react-native/docs/getting-started.html)
+
+There are currently two ways to create a new React Native application;  `react-native-cli` and `create-react-native-app` (`CRNA`). We chose the former because it provides more flexibility when it comes to configuring settings in Xcode and Android Studio. `react-native-cli` also makes it much easier to link third-party libraries to underlying native libraries. 
+
+There is, however, a downside; more flexibility in local configuration can lead to build and runtime errors in Android Studio and Xcode (this is especially apparent in Android Studio with the way it is constantly syncing Gradle files).
+
+> `Project Directories`
+
+You’ll see the `android` and `ios` directories at the root of the project. Both of these can be opened and manipulated in their respective IDE’s.
+
+Note: Never edit Gradle files outside of Android Studio. It’s very easy to create build issues without taking advantage of the sync process embedded in Google’s IDE.
```

Comment:
Maybe phrase this differently:

Note: We recommend that you only edit Gradle files with Android Studio. Google's IDE does a lot of Gradle-related work behind the scenes via a sync process, which makes it very easy to introduce build issues if you modify these files with a different program.